spec/features/users/profile_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "user profile page", type: :feature do
+  describe "As a logged-in user" do
+
+    before :each do
+      @user = User.create(name:"Luke Hunter James-Erickson", address:"123 Lane", city:"Denver", state:"CO", zip:"88888", email: "tombroke@gmail.com", password:"Iamapassword", password_confirmation:"Iamapassword", role: 0)
+ 
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+
```

Comment:
This is the cool method I was talking about for stubbing a logged in user within tests. I think we should start using this as much as possible going forward!

Reply:
@LHJE @priyapower tagging you as well in case you're curious. This method lets us have a logged in user without having to simulate a user clicking login, filling in email/pw, submitting.

Reply:
Yes, good work. I am on board.